docs/useRedirect.md

```diff
@@ -68,4 +65,33 @@ const MyPageButton = () => {
     }
     return <button onClick={handleClick}>My page</button>;
 };
-```
\ No newline at end of file
+```
+
+## Redirect function
+
+`useRedirect` allows you to redirect to the result of a function as follows:
```

Comment:
This deserves more examples as you can also return an object as expected by `useNavigate`